cmd/neofs-cli/modules/accounting/balance.go

```diff
@@ -34,13 +34,12 @@ var accountingBalanceCmd = &cobra.Command{
 
 		balanceOwner, _ := cmd.Flags().GetString(ownerFlag)
 		if balanceOwner == "" {
-			err := user.IDFromSigner(&idUser, neofsecdsa.SignerRFC6979(*pk))
-			common.ExitOnErr(cmd, "decoding user from key", err)
+			idUser.SetScriptHash((*keys.PublicKey)(&pk.PublicKey).GetScriptHash())
```

Comment:
This row is a good candidate to be moved to a separate function inside the `user` package because there are few places in the code, which look too similar

Reply:
yeah but we dropped "user from key" function, so, looks like it needs to be resurrected

Reply:
was added by https://github.com/nspcc-dev/neofs-sdk-go/commit/6ab5d39ba0bf62b31e9423091f508d35cb890ec5, so now i can adopt it